utils/cmut__utils__compile_resource.cmake

```diff
@@ -0,0 +1,139 @@
+
+
+include("${CMAKE_CURRENT_LIST_DIR}/../cmut_message.cmake")
+
+
+function(cmut__utils__get_variable_name VARIABLE_NAME INPUT_FILE)
+    # get the filename and replace all ' ' and '.' with '_'
+    get_filename_component(SOURCE_FILE_NAME ${INPUT_FILE} NAME)
```

Comment:
path variable have to be quoted to avoid error with path with space

Reply:
done